pyanaconda/modules/payloads/payload/dnf/installation.py

```diff
@@ -360,7 +360,7 @@ def run(self):
 
         # Get substitutions for variables.
         # TODO: replace the interpolation with DNF once possible
-        basearch = util.execWithCapture("uname", ["-i"]).strip().replace("'", "")
+        basearch = util.execWithCapture("uname", ["-m"]).strip().replace("'", "")
```

Comment:
I understand this is out of scope for this fix, but this could be simplified by using `os.uname()` or `platform.machine()` instead of running `uname` and parsing its output.

Reply:
Great suggestion - I will incorporate.